### ISSUE TITLE
[feature request] formatter

### ISSUE THREAD

**Issue by jjangga0214** [igorshubovych/markdownlint-cli#59]
In my opinion, it'd be cool to format markdown as like `eslint --fix`. This will silently fix problems if it can, otherwise show error. 

For example, <https://github.com/DavidAnson/vscode-markdownlint> enables this feature and it's super handy. The feature would make workflow more beautiful like below.

```js
{
  "lint-staged": {
    "*.md": [
      "markdownlint --fix",
      "git add"
    ]
  },
}
```

**Comment by DavidAnson** [comment]
I have work planned in the library to enable this.

**Comment by jjangga0214** [comment]
@DavidAnson Happy to see the author of the extension I mentioned! 
Great! Looking forward to use it! :+1:

**Comment by DavidAnson** [comment]
I forgot to update this issue at the time. The `--fix` option was added in version `0.19.0`:
https://github.com/igorshubovych/markdownlint-cli/releases/tag/v0.19.0
